src/_staff/moderation.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Moderation Team
+description: Documents to help guide the Moderation Team.
+---
```

Comment:
This should likely have a small snippet telling the user to choose a document from the sidebar, its not immediately obvious its changed when navigating to this page.

Reply:
I agree, I was planning on filling out this page (and the other one) after merge, but it can't hurt to add something.

Reply:
Alright, I've added some text and will get the test site updated shortly o7